life_hc/MySQL/Housing/fn_houseCleanup.sqf

```diff
@@ -7,15 +7,14 @@
     Description:
     Cleans up containers inside in house of player.
 */
-private ["_query","_containers"];
-_query = format ["SELECT pid, pos FROM containers WHERE pid='%1' AND owned='1'",_this];
 
-_containers = [_query,2,true] call HC_fnc_asyncCall;
+private _query = format ["selectContainerPositions:%1", _this];
+private _containers = [_query, 2, true] call HC_fnc_asyncCall;
 
 if (_containers isEqualTo []) exitWith {};
 {
-    _pos = call compile format ["%1",_x select 1];
-    _container = nearestObjects[_pos,["Box_IND_Grenades_F","B_supplyCrate_F"],12];
+    _pos = call compile format ["%1", _x select 1];
```

Comment:
All this does is a simple `missionNamespace getVariable` just that call compile format is a fuckton slower and quite dumb way to do this.